src/plugins/field_formats/common/constants/color_default.ts

```diff
@@ -12,4 +12,5 @@ export const DEFAULT_CONVERTER_COLOR = {
   regex: '<insert regex>',
   text: '#000000',
   background: '#ffffff',
+  boolean: 'true or false',
```

Comment:
```suggestion
  boolean: 'true',
```
We'd also need to update the default value if we go with the select menu instead.